gulpfile.mjs

```diff
@@ -688,7 +688,12 @@ function createTestSource(testsName, { bot = false, xfaOnly = false } = {}) {
     if (process.argv.includes("--headless")) {
       args.push("--headless");
     }
-
+    if (process.argv.includes("--numSlaves")) {
+      args.push(
+        "--numSlaves",
+        process.argv[process.argv.indexOf("--numSlaves") + 1]
+      );
+    }
```

Comment:
Please let's not lose the existing blank-line here, since it aids in readability.
```suggestion
    }
    
```